db/data/20140823134218_set_basepath_where_nil.rb

```diff
@@ -0,0 +1,14 @@
+class SetBasepathWhereNil < ActiveRecord::Migration
+  def self.up
+    OntologyVersion.where(basepath: nil).find_each do |ov|
```

Comment:
With `select` **and** `includes`, the columns don't get updated (no matter in what order I use those two methods). So I just use `includes` in 232f6a3.